src/Cassandra/ProtocolEvents/DotnetTimer.cs

```diff
@@ -0,0 +1,47 @@
+﻿//
+//       Copyright (C) DataStax Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+
+using System;
+using System.Threading;
+
+namespace Cassandra.ProtocolEvents
+{
+    /// <summary>
+    /// <para><inheritdoc /></para>
```

Comment:
NIT: I think `inheritdoc` is a root level element and can't be nested within `para`.

Reply:
The enhanced tooltip extension that I have is rendering this correctly but I'll generate logs to see what the result is.

Reply:
Nevermind, the api docs only show the `public` API so all it matters is the tooltip for the driver devs and on my tooltip it's working so I guess it's fine?

Reply:
no problem, nvm.